app/javascript/chat/ActiveChatChannel/ChatMessages.js

```diff
@@ -0,0 +1,73 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import ActionMessage from '../actionMessage';
+import Message from '../message';
+import DirectChatInfoMessage from './IntroductionMessages/DireactIntoMessage';
+import OpenChatInfoMessagge from './IntroductionMessages/OpenIntroMessage';
+
+const ChatMessages = ({
+  activeChannelId,
+  messages,
+  showTimestamp,
+  activeChannel,
+  currentUserId,
+  triggerActiveContent,
+  triggerEditMessage,
+  triggerDeleteMessage,
+}) => {
+  if (!messages[activeChannelId]) {
+    return '';
```

Comment:
refactor (best practice): There is no need to return an empty string. If a component ends up returning nothing via conditional rendering, return `null`.